### ISSUE TITLE
Face recognition for high accuracy

### ISSUE THREAD

**Issue by roiksail** [ageitgey/face_recognition#448]
Hello everybody...i maked this project using raspberry pi 3 in real time..i added 45 people and In some people they identify faces incorrectly,how i can to increase face recognition accuracy??

**Comment by roiksail** [comment]
How do you use multiple photos for each person?i used facerec_from_webcam_faster.py in the examples directory

**Comment by stoneWeb** [comment]
Same question. How do you use multiple photos for each person? I used opencv demo.

**Comment by xugaoxiang** [comment]
@roiksail @stoneWeb , I think, @cftang0827  just use more photos for a person on training.

**Comment by cftang0827** [comment]
Yes, I used scikit-learn SVM and KNN module to make a simple classifier. However, the cons is that you can only classify the person in your database.

[Person1] vector1
[Person1] vector2
[Person1] vector3

[Person2] vector1
[Person2] vector2
[Person2] vector3
[Person2] vector4

For my experience, I think knn with distance metric is better.
And you may use svm for gender classification

Thanks.
